manuscript_heb/00-Introduction.md

```diff
@@ -2,171 +2,155 @@
-לכן , יתכן שבסביבות מסויימות תתקלו בבעיות תאימות לתקן שלא נכללות בספר זה. 
- 
-### למי מיועד ספר זה 
+###תאימות ל Node.js ודפדפנים 
 
-ספר זה נועד להדריך אלו המכירים  JavaScript  ו ECMAScript 5.  בעוד שהבנת JavaScript לעומק אינה הכרחית לשימוש בספר זה ,  הוא יסיעע לכם להבין ההבדלים בין   ECMAScript 5  ו ECMAScript 6. 
-במיוחד , ספר זה נועד  למעבר  מהיר לתכנות מתקדם  למפתחים בסביבת   Node.JS וקוד לדפדפנים המעונינים ללמוד על התפתחויות  עדכניות  בשפה וליישם אותם. 
-ספר זה אינו למתחילים שמעולם לא תכנתו ב  JavaScript. הנכם אמורים להיות בעלי הבנה בסיסית טובה  בשפה בכדי להשתמש  בספר זה. 
- 
+סביבות רבות של JavaScript - הן בדפדפנים שונים והן ב-Node.js , עובדות על יישום בפועל של ECMAScript 6.
+ספר זה אינו עוסק בבעיות יישום התקן בסביבות השונות אלא מתרכז בהגדרות התקן והתנהגות נכונה.
+לכן - ייתכן שבסביבות מסויימות תתקלו בבעיות תאימות לתקן, אשר אינן נכללות בספר זה.
  
-###  תוכן ענינים 
-
-כל אחד מ - 13 הפרקים בספר מכסה  אספקט אחר  ב   ECMAScript 6.  פרקים רבים מתחילים בדיון על  הבעיות ש  ECMAScript 6 נועדו לפתור ,   בכדי להציג יריעה רחבה לסיבות  ששינוים אלו הוגדרו.   
-כל פרק מכיל מספר דוגמאות קוד לסייע לך ללמוד סינטאקס חדש וקונספטים חדשים. 
-
+###למי מיועד ספר זה 
 
-<B>פרק  1 : כיצד עובד Block Binding </B>  הפרק מדבר על    `let` and `const` , החלפת block-level gcur  var. 
+ספר זה נועד להדריך את אלו המכירים JavaScript ו-ECMAScript 5. בעוד שהבנת JavaScript לעומק אינה הכרחית לשימוש בספר זה, הוא יסייע לכם להבין את ההבדלים בין ECMAScript 5 ובין ECMAScript 6.
+ספר זה נועד למעבר מהיר לתכנות מתקדם עבור מפתחים בסביבת Node.JS וקוד לדפדפנים, אשר מעוניינים ללמוד על התפתחויות עדכניות בשפה וליישם אותן.
+ספר זה אינו למתחילים שמעולם לא תכנתו ב JavaScript. הנכם אמורים להיות בעלי הבנה בסיסית טובה בשפה בכדי להשתמש בספר זה. 
+ 
+###תוכן עניינים 
 
-<B> פרק 2 : Strings ו Regular Expressions </B> הפרק סוקר פונקציונאליות נוספת חדשה לעיבוד ומניפולאציות על מחרוזות string ו בחינה  וכן מציג הקונספט החדש  template strings. 
+כל אחד מ-13 הפרקים בספר מכסה אספקט אחר ב-ECMAScript 6. פרקים רבים מתחילים בדיון על הבעיות ש-ECMAScript 6 נועדה לפתור, על מנת להציג יריעה רחבה לסיבות לשינויים אלו.
+כל פרק מכיל מספר דוגמאות קוד על מנת לסייע לך ללמוד מבנה חדש בשפה וקונספטים תכנותיים חדשים.
 
- 
-<B> פרק 3 : Functions in ECMAScript 6 </B>  הפרק עוסק בשינויים לגבי פונקציות.   זה כולל  תבנית פונקציות חץ,  ערך ברירת מחדל לפרמטרים של פונקציה  ונושאים נוספים. 
-  
-<B>   פרק 4 :  הרחבת פונקציונאליות של אוביקטים   </B>  מסביר את השינוי כיצד אוביקטים נוצרים, עוברים שינוי וכיצד משתמשים בהם.  הנושאים כוללים שינוי בליטרל של אוביקט , ומטודות הצגה. 
+<B>פרק 1 : כיצד עובד Block Binding </B> מדבר על `let` ו-`const` - החלפת block-level ב-var.
 
-<B>  פרק 5 : פעולת הרס אוביקטים  עבור גישה פשוטה יותר לנתונים </B>  מציג הרס אוביקטים ומערכים , המאפשר לבמע דקומפוז  של אוביקטים ומערכים בצורת כתיבה קצרה יותר. 
+<B> פרק 2 : Strings ו Regular Expressions </B> סוקר יכולת נוספת חדשה לעיבוד על מחרוזות וכן מציג את הקונספט החדש של template strings.
```

Comment:
I don't think we should have any issue translating "template strings"

Reply:
איך כדאי לתרגם?

מחרוזת תבנית?
מחרוזת טמפלייט?
טמפלייט סטרינג?